pkg/base/config.go

```diff
@@ -143,6 +143,10 @@ const (
 	// DefaultDescriptorLeaseRenewalTimeout is the default time
 	// before a lease expires when acquisition to renew the lease begins.
 	DefaultDescriptorLeaseRenewalTimeout = time.Minute
+
+	// DefaultDescriptorLeaseRenewalTimeout is the default time
+	// before a lease expires when acquisition to renew the lease begins.
```

Comment:
Do we want this to default to false?

Reply:
Let me change the default here.

Reply:
I'm fine with changing the default on master.

Reply:
Up to you

Reply:
For consistency I'll just leave this as enabled, we can turn it back on after your change.